src/repos/UserSessionRepo.ts

```diff
@@ -23,8 +21,8 @@ const createOrUpdateSession = async (
 ): Promise<UserSession> => {
   let session = await db()
     .createQueryBuilder('usersessions')
-    .innerJoin('usersessions.user', 'user', 'user.uuid = :userID')
-    .setParameters({ userID: user.id })
+    .innerJoin('usersessions.user', 'user', 'user.netID = :userNetID')
+    .setParameters({ userNetID: user.netID })
```

Comment:
hm why is this not just NetID: user.netID

Reply:
I think we can keep this naming scheme for now just because userNetID indicates the netID is coming from the user argument.